blocks/v2-video/v2-video.js

```diff
@@ -54,7 +54,7 @@ export default async function decorate(block) {
   content.classList.add(`${blockClass}__content`);
   [...headings].forEach((heading) => heading.classList.add(`${blockClass}__heading`));
   [...ctaButtons].forEach((button) => {
-    button.classList.add(`${blockClass}__button`, 'tertiary');
```

Comment:
Hover state is not correct:
<img width="1074" alt="image" src="https://github.com/Netcentric/vg-volvotrucks-us-rd/assets/3090999/8711ee1b-c4d6-44a9-8213-2884ac61e7fe">